### ISSUE TITLE
Help Command does not add a reaction to the message

### ISSUE THREAD

**Issue by Ma-wa-re** [strinking/futaba#63]
Since the help command is a built-in command to the bot extension of discord.py it dose not add a reaction like the rest of the commands. Therefore we need to find a way to add that reaction to the message.
If possible we should make it add the reaction to any command at bot level meaning it does not require the user developing the cog to add it to each command

**Comment by Ma-wa-re** [comment]
From looking at discord.py docs we can use the [on_command_completion](https://discordpy.readthedocs.io/en/rewrite/ext/commands/api.html#discord.on_command_completion) this will be able to add the reaction is the command completed successfully for any command meaning the developer of a cog does not require to add the reaction on every command (We will create another issue for removing the current reactions in commands)

**Comment by ammongit** [comment]
#27 should be completed first so that we can add a special Exception for command errors, complete with optional information `content`/`embed` information to send about the errors, for :x: results.
